LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2017 fh1ch
+Copyright (c) 2017 Fabio Huser <fabio@fh1.ch>
```

Comment:
Good point 👍 

The`README.md` already uses the correct copyright information (see https://github.com/fh1ch/node-bacstack/blob/master/README.md#license)

Reply:
yep, I know